wuxiaworld2ebook.py

```diff
@@ -117,9 +131,16 @@ def button_press():
 ending_chapter_chosen = ttk.Entry(app, width = 5, textvariable = ending_chapter)
 ending_chapter_chosen.grid(column = 1, row = 3, sticky = "W")
 
+#Code for delete chapters
+label5 = ttk.Label(app, text = "Cleanup when done: ")
+label5.grid(column = 0, row = 4, pady = 10, sticky = "W")
+delete_chapters = tk.BooleanVar()
```

Comment:
Can you change the default value of `delete_chapters` to True, so the default behaviour still deletes the temporary files?

Reply:
I'm not a fan of having all the chapters deleted  by default, but it is indeed a breaking change, so I added a config.json file with a boolean that is read when teh program launches so the user can configure it at will.

Main reason I'm not a fan of the deletion by default is because 1, it stresses wuxia servers if the user created a book from chapter 10 to 879 has has to restart to have 1-879, and 2, because it can take a while to get all the chapters. I think it took 20 minutes for me to get all the currently released Overgeared chapters.

This ties into the fact that I think it could be cool to have the option to split by "volume" , which could let the user say to split into parts of 300 chapters each. Allowing them to keep the temp chapters would let them "complete" a volume once more chapters come out. 

Of course, by keeping a cache of chapter you lose the chance of getting a "revised" chapter with a typo correction and whatnot, but such revisions are probably so infrequent people wont care.

Reply:
Hm, I think I never downloaded a chapter twice except for testing. I just assumed that this would be only practical for a few people and clutter up the harddrive for the rest.

But the config file should be the most optimal solution.